Deploy_online_101.md

```diff
@@ -7,8 +7,26 @@ title: Display Data
 
 > Nelisa wants to see her statistics on a web page from her smartphone.
 
+The Heroku deployment process is built on top of git. To deploy to ir you need to add a Heroku git repository. The Heroku deployment process will guide you through the required deployment steps.
+
+**Please note:**
+
+* You applications `package.json` file's `dependencies` sections should contain all the modules your application depends on. So that Heroku knows which modules to install. Installing modules using `npm --save <module_name>` will keep your package.json up to date.
```

Comment:
The `dependencies` section of your application's `package.json` file should contain all the modules your application depends on; Heroku will use it install your modules. When you're developing your application, installing modules using `npm --save <module_name>`. This adds the module(s) to your `package.json` so that it's kept up to date.